gammacat/collection.py

```diff
@@ -227,8 +270,26 @@ def make_index_file_for_output(self):
             resource = SED.read(self.config.path / filename).resource
             resource.location = filename
             resources.append(resource)
+        for filename in self.config.lc_files():
+            resource = LightCurve.read(self.config.path / filename).resource
+            resource.location = filename
+            resources.append(resource)
+        for filename in self.config.ds_files():
+            resource = DataSet(self.config.path / filename).to_GammaCatResource()
+            resource.location = filename
+            resources.append(resource)
+
+        for filename in self.config.lc_files():
+            resource = LightCurve.read(self.config.path / filename).resource
+            resource.location = filename
+            resources.append(resource)
+        for filename in self.config.ds_files():
+            resource = DataSet(self.config.path / filename).resource()
```

Comment:
I think here you need to remove the `()` at the end of the line because this is a property, no?